subsys/fs/nffs_fs.c

```diff
@@ -19,6 +19,18 @@
 
 #define NFFS_MAX_FILE_NAME 256
 
+#if USE_PARTITION_MANAGER
+#include <pm_config.h>
+#define NFFS_FLASH_AREA_STORAGE_OFFSET PM_MCUBOOT_STORAGE_ADDRESS
```

Comment:
I'm a bit unsure if this is correct, since you could have the use case where you would use NFFS in your application. This should probably be changed to something more image specific?

Reply:
The semantics mirror that of DT right? If so there will be no regression and IMO this should be OK.

Reply:
@hakonfam  Is that possible that storage partition is not described via mcuboot pm.yaml, while it is well described via application dts? (so lack of PM_MCUBOOT_STORAGE_ADDRESS macro). Does it mak sense along witch PM usage.

Reply:
the storage partition is specified by pm.yml, so pm_config.h should have a define for that the same way that the app dts has it.

Reply:
I'm just wondering if maybe we could reuse some of the `#define FLASH_MAP_OFFSET(i) UTIL_CAT(PM_, UTIL_CAT(PM_##i##_LABEL, _ADDRESS))` macro to give `STORAGE_ADDRESS` etc if the **application** and **mcuboot** both have their *own* file system for storage? I don't know if this is a valid use-case.

Reply:
Currently I don't see need for storage partition for mcuboot.